cmd/qosserver/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	log "github.com/sirupsen/logrus"
+)
+
+var QoSSuccessfulServerRequests = promauto.NewCounterVec(prometheus.CounterOpts{
+	Namespace: "thundernetes",
+	Name:      "qos_successful_server_requests",
+	Help:      "Number of requests made to the QoS server",
+}, []string{"remote_ip"})
+
+func main() {
+	go metricsServer(3075)
```

Comment:
We should avoid hard-coding the ports in code. Can this be provided via config or ENV, with a smart default? This allows someone to alter the port without having to recompile the code.

Reply:
Thanks Ken, I'll change the way the port is handled. Do you think the port for the prometheus metrics should also be configurable?